test/test-applications/throughput/Samples.AspNetCoreSimpleController/Samples.AspNetCoreSimpleController.csproj

```diff
@@ -5,4 +5,8 @@
     <Platforms>x64;x86;AnyCPU</Platforms>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Why is this needed to be referenced by the samples now?

Reply:
I guess there was a rebase error at some point but it couldnt compile because of Program.cs line 12, using the Instrumentation type, this reference is on master..